src/main/java/org/newrelic/nrjmx/Arguments.java

```diff
@@ -88,6 +93,7 @@ static Arguments from(String[] args) throws ParseException {
         argsObj.help = cmd.hasOption("help");
         argsObj.debug = cmd.hasOption("debug");
         argsObj.isRemoteJMX = cmd.hasOption("remote");
+        argsObj.isRemoteJBossStandalone = cmd.hasOption("remoteJBossStandalone");
```

Comment:
we should put string literals as constants, to avoid mistakes on modifying remoteJBossStandalone here and not in the Option construction.